### ISSUE TITLE
Review new GPU and OpenCL documentation (Trac #734)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#850]
Migrated from http://trac.sasview.org/ticket/734
```json
{
    "status": "closed",
    "changetime": "2016-10-08T15:47:07",
    "_ts": "2016-10-08 15:47:07.634683+00:00",
    "description": "",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-10-08T15:38:07",
    "component": "SasView",
    "summary": "Review new GPU and OpenCL documentation",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "smk78",
    "type": "task"
}
```

**Comment by smk78** [comment]
Trac update at `2016/10/08 15:45:00`: **smk78** commented:

In changeset 711f17e515fff416ab1aa15ae93c3d245ff21525:

```text
#!CommitTicketReference repository="sasmodels" revision="711f17e515fff416ab1aa15ae93c3d245ff21525"
Tweaks to text. Operation of python shell device test verified. Refs
#850
```

**Comment by smk78** [comment]
Trac update at `2016/10/08 15:47:07`: 
* **smk78** commented:

In changeset ec8b9a3e2841d6f1ececef27cda299f684cf8285:

```text
#!CommitTicketReference repository="sasmodels" revision="ec8b9a3e2841d6f1ececef27cda299f684cf8285"
Tweaks to text, including a warning about OpenCL drivers and a link to
an Apple compatibility table. Closes #850
```


* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"
